app/build.gradle

```diff
@@ -149,6 +160,9 @@ configurations {
     geckoNightlyAarch64Implementation {}
 }
 
+def buildconfig = new Yaml().load(new File(rootDir, '.buildconfig.yml').newInputStream())
+def gecko = new Gecko(buildconfig.geckoNightlyVersion)
```

Comment:
I'm just doing this straight in `build.gradle` since it isn't shared between different gradle projects.
Once there's multiple gradle modules that all need shared dependency config, I'd recommend throwing this on `gradle.rootProject.ext` so that all modules can reuse the loaded file